### ISSUE TITLE
Handling of multiple rotamers in PDB files

### ISSUE THREAD

**Issue by lucasmiranda42** [BorgwardtLab/proteinshake#270]
Hi! 

Upon working with `ProteinFamilyDataset`, I ran into some issues due to multiple rotamers being present in the loaded PDB files.

First, and just in case, my code to load the data:

```python
from proteinshake.tasks.pfam_task import ProteinFamilyDataset
pfam_dataset=ProteinFamilyDataset(only_single_chain=True, use_precomputed=True)
```

When I iterate through and process the downloaded proteins, my goal is to keep coordinates for each `CA`, `C`, `O`, and `N` atoms only. Upon loading some structures, such as [this one](https://www.rcsb.org/3d-view/3U3U), I see that some residues (in this case ASP11) have duplicated coordinates corresponding to multiple rotational conformations of the side chain:

<img width="808" alt="image" src="https://github.com/BorgwardtLab/proteinshake/assets/45250053/8d8702fe-d6bc-4ad5-a874-a82a1479dde8">

In the raw PDB file, this is coded in the name of the residue. The alpha carbon of residue 11 is, for example, tagged both as `AASP` and as `BASP`.

```
ATOM     82  CA AASP C  11      14.656   6.921 -15.720  0.42 37.13           C  
ANISOU   82  CA AASP C  11     4826   5151   4130    771   -213   -129       C  
ATOM     83  CA BASP C  11      14.654   6.917 -15.721  0.58 37.11           C  
ANISOU   83  CA BASP C  11     4824   5150   4128    771   -214   -130       C  
```

Would it be possible to select and handle rotamers within proteinshake directly? In my current application, I can get away with selecting any conformation in these simple cases, but this may not always be applicable.

Best, and thank you!
Lucas

**Comment by timkucera** [comment]
Good catch and great issue. Programmatically there would be multiple ways to deal with this, but to keep things simple I would probably remove those in the preprocessing (unless there is a good reason not to?). I will keep it in mind for the upcoming release.

**Comment by pjhartout** [comment]
@timkucera beyond rotamers, some structures have multiple states (e.g. apo/holo states for proteins with ligand binders, or other complex proteins, e.g. https://www.rcsb.org/structure/5are).
